cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/mongo/MongoBackend.java

```diff
@@ -66,42 +67,45 @@ void createCollection(String dbName, String collectionName, long collectionsSize
      * @throws Exception
      */
     void insertContextDataRaw(String dbName, String collectionName, ArrayList<Document> aggregation) throws Exception;
-
+    
     /**
-     * Inserts a new document in the given aggregated collection within the given database (row-like mode).
+     * Inserts pre-aggregated numeric context data in the related aggregated collction for the resolutions given.
      * @param dbName
      * @param collectionName
      * @param recvTimeTs
      * @param entityId
      * @param entityType
      * @param attrName
      * @param attrType
-     * @param attrValue
-     * @param attrMd
+     * @param max
+     * @param min
+     * @param sum
      * @param resolutions
+     * @param numSamples
+     * @param sum2
      * @throws Exception
      */
-    void insertContextDataAggregated(String dbName, String collectionName, long recvTimeTs,
-            String entityId, String entityType, String attrName, String attrType, String attrValue, String attrMd,
-            boolean[] resolutions)
+    void insertContextDataAggregated(String dbName, String collectionName, long recvTimeTs, String entityId,
+            String entityType, String attrName, String attrType, double max, double min, double sum, double sum2,
+            int numSamples, boolean[] resolutions)
         throws Exception;
-
+    
     /**
-     * Stores in per-service/database "collection_names" collection the matching between a hash and the fields used to
-     * build it.
-     * FIXME: destination is under study
+     * Inserts pre-aggregated string-based context data in the related aggregated collction for the resolutions given.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1426/commits/a3bb342105b0c86818d833b69735985f450f02f8